pkg/apis/serving/v1alpha3/service.go

```diff
@@ -64,11 +64,14 @@ type ComponentExtensionSpec struct {
 	// Maximum number of replicas for autoscaling.
 	// +optional
 	MaxReplicas int `json:"maxReplicas,omitempty"`
-	// Parallelism specifies how many requests can be processed concurrently, this sets the target
+	// Concurrency specifies how many requests can be processed concurrently, this sets the target
 	// concurrency for Autoscaling(KPA). For model servers that support tuning parallelism will use this value,
 	// by default the parallelism is the number of the CPU cores for most of the model servers.
```

Comment:
"by default the parallelism is the number of CPU cores" makes it sound like not setting this value will default it to number of CPU cores in the predictor. I don't think that is true?